plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerUtility.java

```diff
@@ -17,17 +17,17 @@
 	public static HashMap<String, String> parseLines(String[] lines) {
 		HashMap<String, String> adbEnv = new HashMap<>();
 
-		String setEnvWin = "setx ";
```

Comment:
I checked that on my Win10 box CDKv2 vagrant service-manager env returns the following:

    > vagrant service-manager env                                                                                                  
                                                                                                                               
    # docker env:                                                                                                                  
    # Set the following environment variables to enable access to the                                                              
    # docker daemon running inside of the vagrant virtual machine:                                                                 
    export DOCKER_HOST=tcp://10.1.2.2:2376                                                                                         
    export DOCKER_CERT_PATH='C:\Users\JeffMAURY\CDK\2.3\components\rhel\rhel-    ose\.vagrant\machines\default\virtualbox\docker\'     
    export DOCKER_TLS_VERIFY=1                                                                                                     
    export DOCKER_API_VERSION=1.22                                                                                                 
                                                                                                                               
                                                                                                                               
    # openshift env:                                                                                                               
    # You can access the OpenShift console on: https://10.1.2.2:8443/console                                                       
    # To use OpenShift CLI, run: oc login https://10.1.2.2:8443                                                                    
    export OPENSHIFT_URL=https://10.1.2.2:8443                                                                                     
    export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console                                                                     
    export DOCKER_REGISTRY=hub.openshift.rhel-cdk.10.1.2.2.xip.io                                                                  
                                                                                                                               
    # run following command to configure your shell:                                                                               
    # eval "$(VAGRANT_NO_COLOR=1 vagrant service-manager env | tr -d '\r')"                                                        
                                                                                                                               
At least, the CDKv3 on Windows is using the SET notation:

    > minishift docker-env                                             
    SET DOCKER_TLS_VERIFY=1                                            
    SET DOCKER_HOST=tcp://192.168.99.104:2376                          
    SET DOCKER_CERT_PATH=C:\Users\JeffMAURY\.minishift\certs           
    SET DOCKER_API_VERSION=1.23                                        
    REM Run this command to configure your shell:                      
    REM     @FOR /f "tokens=*" %i IN ('minishift docker-env') DO @%i   

so the current code does not on Windows. I'm gonna check if I can make CDKv2 issue setx statements on Windows